lib/constants.js

```diff
@@ -1,4 +1,6 @@
 module.exports = {
+  promptMaxChoices: 12,
```

Comment:
Why a max now? How will we see more than 12?
Personally, I would delay addition of this until a) it is a problem, or b) we can handle more than 12 and not just ignore them

Reply:
the default max is lower than the number of inputs, 12 was an arbitrary number (some number to delay the problem 😉 )
I've changed this to the size of inputs

Reply:
also the max here is not a hardlimit, inquirer will still let you navigate through all choices even if you have 100